functions/interface.php

```diff
@@ -107,7 +106,7 @@ function admin_notices() {
 
 				<?php foreach ( $notices['server_config'] as $notice ) : ?>
 
-					<li><?php echo wp_kses_data( $notice ); ?></li>
+					<li><?php echo print_whitelist_html( $notice ); ?></li>
```

Comment:
@rmccue so I've got a question here. Below in this file, I'm storing translated strings in vars, each has has it's own let of allowed HTML and some don't have allowed HTML. So far so good. However, what do I use here as a generic escaping function to display text with varied set of allowed HTML. Does it make sense?

To adhere to `escape late` WP rule, here must be some escape function here. But then, how to I cater for strings with varied set of allowed HTML? In your [Whitelist HTML docs](https://github.com/humanmade/whitelist-html) there is example how to save sage HTML to a variable, but there is no example how to correctly echo it afterwards. It would be great if you could add it to the docs :)

Thanks!